### PR TITLE
refactor(core/delivery): Convert execution filters to React

### PR DESCRIPTION
**Pull request by jrsquared**

### CODE REVIEW

File: app/scripts/modules/core/src/cluster/filter/collapsibleFilterSection.directive.js [hunk 1]
```diff
@@ -24,50 +24,6 @@ module.exports = angular
...
       },
-
-      controller: function($scope) {
-        $scope.$on('parent::toggle', function(event, isShow) {
-          $scope.state = {expanded: isShow };
-        });
-      }
-    };
-  })
-  .directive('filterToggleAll', function () {
```
Comment (jrsquared) [inline app/scripts/modules/core/src/cluster/filter/collapsibleFilterSection.directive.js]: Couldn't find any uses of it anywhere

File: app/scripts/modules/core/src/delivery/filter/ExecutionFilters.tsx [hunk 2]
```diff
@@ -0,0 +1,287 @@
...
+    const { application } = this.props;
+    const { $rootScope, executionFilterModel, executionFilterService } = ReactInjector;
+
+    this.executionsRefreshUnsubscribe = application.executions.onRefresh(null, () => { this.refreshPipelines(); });
+    this.groupsUpdatedSubscription = executionFilterService.groupsUpdatedStream.subscribe(() => this.setState({ tags: executionFilterModel.asFilterModel.tags }));
+    this.pipelineConfigsRefreshUnsubscribe = application.pipelineConfigs.onRefresh(null, () => { this.refreshPipelines(); });
+
+    this.initialize();
+
+    this.locationChangeUnsubscribe = $rootScope.$on('$locationChangeSuccess', () => {
```
Comment (anotherchrisberry) [inline app/scripts/modules/core/src/delivery/filter/ExecutionFilters.tsx]: would check with @christopherthielen if there's a better way to handle this in react world

File: app/scripts/modules/core/src/delivery/filter/ExecutionFilters.tsx [hunk 3]
```diff
@@ -0,0 +1,287 @@
...
+  return (
+    <div className="checkbox">
+      <label>
+        <input type="checkbox" checked={sortFilter.status[props.status] || false} onChange={changed}/>
+        {props.label}
+      </label>
+    </div>
+  );
+}
+
```
Comment (anotherchrisberry) [inline app/scripts/modules/core/src/delivery/filter/ExecutionFilters.tsx]: not helpful

File: app/scripts/modules/core/src/delivery/filter/executionFilters.less [hunk 4]
```diff
@@ -1,12 +1,13 @@
...
       display: inline-block;
       margin-left: -14px;
+      margin-right: 3px;
       opacity: 0.7;
       font-size: 80%;
     }
   }
-}
\ No newline at end of file
+}
```
Comment (anotherchrisberry) [inline app/scripts/modules/core/src/delivery/filter/executionFilters.less]: helpful

**Comment by anotherchrisberry** [comment]
pretty much seems fine

**Comment by anotherchrisberry** [comment]
The best way to test this is live, merge whenever you're comfortable with it

**Pull request merged**